core/src/main/java/org/apache/calcite/sql/fun/SqlQuantifyOperator.java

```diff
@@ -160,4 +172,23 @@ public class SqlQuantifyOperator extends SqlInOperator {
       throw new AssertionError("unexpected " + kind);
     }
   }
+
+  @Override public void unparse(
```

Comment:
is there a test for this new function?

Reply:
1. `testAllListWithOneElement` 
2. `SqlOperatorUnparseTest#testQuantifyCollectionOperators2`
These two can cover this test case.

Reply:
Changing the Parser.jj parsing will complicate this issue, so I have reverted this part of the modification. The corrections will be made in a new PR later.